src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -61,6 +65,15 @@ public void setupRootModel(ModifiableRootModel model) throws ConfigurationExcept
   }
 
   @Override
+  public ModuleWizardStep modifyStep(SettingsStep step) {
+    if (getModuleType() != null && step instanceof ProjectSettingsStep) {
```

Comment:
Project Wizard paths are completely different in different IDEs.

See the location of the `ProjectSettingsStep.java` file in the repository to get a hint ;)